integrations/slack.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by idoschachter on 17/07/2016.
+ */
+'use strict';
+
+const Promise = require('bluebird');
+const Slack = require('node-slack');
+
+const hookUrl = process.env.SLACK_URL || null;
```

Comment:
Yes, I will

On Sunday, July 17, 2016, Nitai J. Perez <notifications@github.com> wrote:

> In integrations/slack.js
> <https://github.com/CommonRaven/PokePush/pull/1#discussion_r71085301>:
>
> > @@ -0,0 +1,53 @@
> > +/**
> > + * Created by idoschachter on 17/07/2016.
> > + */
> > +'use strict';
> > +
> > +const Promise = require('bluebird');
> > +const Slack = require('node-slack');
> > +
> > +const hookUrl = process.env.SLACK_URL || null;
>
> Update readme on usage?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CommonRaven/PokePush/pull/1/files/4a31df91ea8ee168ed86f0a56d370d7abcb50a4c#r71085301>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFl9REvbYBFkH2Gm5-VAXJs2JJESaewRks5qWlqngaJpZM4JOQO5>
> .
>


-- 
Ido Schachter
Software developer
[image: ironSource] <http://www.ironsrc.com>

email ido.schachter@ironsrc.com
mobile +972-52-605-2282
fax +972-77-5448273
28 Lilienblum St., Tel Aviv
ironsrc.com <http://www.ironsrc.com>
[image: linkedin] <https://www.linkedin.com/company/ironsource> [image:
twitter] <https://twitter.com/ironsource> [image: facebook]
<https://www.facebook.com/ironSource> [image: googleplus]
<https://plus.google.com/+ironsrc>
This email (including any attachments) is for the sole use of the intended
recipient and may contain confidential information which may be protected
by legal privilege. If you are not the intended recipient, or the employee
or agent responsible for delivering it to the intended recipient, you are
hereby notified that any use, dissemination, distribution or copying of
this communication and/or its content is strictly prohibited. If you are
not the intended recipient, please immediately notify us by reply email or
by telephone, delete this email and destroy any copies. Thank you.